shipcloud.gemspec

```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
+# frozen_string_literal: true
```

Comment:
```suggestion
# frozen_string_literal: true

```
all the other files have an empty following the magic comment, so we could do this here as well.